.gitignore

```diff
@@ -14,16 +14,25 @@
 ##########
 # Addons files
 ##########
+!/addons
 
 # the main plugin
+!/addons/dialog_plugin/
 !/addons/dialog_plugin/*
 
 # the unit-test plugin that needs an update
+!/addons/WAT/
 !/addons/WAT/*
+!/tests/
 !/tests/*
 
 
 ##########
 # Docs
 ##########
-!/docs/*
\ No newline at end of file
+!/docs/
+!/docs/*
+
+
+# But at the end of it all, re-ignore .import files
+*.import
```

Comment:
I can't ignore `.import` files, just the `.import/` folder. 

`.import` files are the instructions to import a file in a certain way, generated by Godot with the default template when there's no import file, or used by Godot to know how to handle a certain file.

Reply:
Ah, I thought you intended to ignore .import files since they are not present in the repository.